scss/atoms/notifications/_notifications.scss

```diff
@@ -0,0 +1,34 @@
+/*
+* ==========================================================================
+* Notifications
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
+	"mixins";
+
+.notification {
+	@include notification(default);
+}
+
+.notification-default {
```

Comment:
You can simplify writing this by using `&-default` underneath the `.notification` class.  Not sure if we want to organize things that way, but it would prevent having to type "notification" over and over again.